csharp/Adapter/Microsoft.Spark.CSharp/Core/PipelinedRDD.cs

```diff
@@ -39,13 +39,12 @@ public override RDD<U1> MapPartitionsWithIndex<U1>(Func<int, IEnumerable<U>, IEn
                 {
                     workerFunc = newWorkerFunc,
                     preservesPartitioning = preservesPartitioning && preservesPartitioningParam,
-                    previousRddProxy = this.previousRddProxy,
-                    prevSerializedMode = this.prevSerializedMode,
-
-                    sparkContext = this.sparkContext,
```

Comment:
"this." prefix seems to me helps readability - we may want to keep "this." prefix in this specific case.